index.html

```diff
@@ -45,6 +45,224 @@ <h1 class="name">I’m Ali Hassan. Glad to see you!</h1>
+                  <li class="tags">
+                    <a class="tag-btn" href="#">javaScript</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="project-button-container">
+                <a href="#" class="project-button">See project</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="content work-projects work-projects-b">
+          <div class="item project-header">
+            <div class="project-img-container_1">
+              <img src="./images/work/img_4.PNG" alt="" class="full-image" />
+            </div>
+          </div>
+          <div class="card-content">
+            <div class="item project-description">
+              <div class="project-title-container">
+                <h4 class="description-title">
+                  Multi-Post <br />
+                  Stories
+                </h4>
+              </div>
+              <div class="project-subtitle-container">
+                <ul class="subtitle-items">
+                  <li class="list-less"><p>CANOPY</p></li>
+                  <li class="list"><p>Back End Dev</p></li>
+                  <li class="list"><p>2015</p></li>
+                </ul>
+              </div>
+              <div class="project-article-container">
+                <p class="description-item description-item-b">
+                  A daily selection of privately perosonalized reads; no
+                  accounts or sign-ups required.
+                </p>
+              </div>
+              <div class="project-tag-container">
+                <ul class="tag-item">
+                  <li class="tags"><a class="tag-btn" href="#">html</a></li>
+                  <li class="tags"><a class="tag-btn" href="#">css</a></li>
+                  <li class="tags">
+                    <a class="tag-btn" href="#">javaScript</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="project-button-container">
+                <input
+                  type="button"
+                  value="See Project"
+                  class="project-button" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="about-myself">
+        <div class="about-myself">
+          <div class="name-container-1">
+            <h2 class="name">About Myself</h2>
+          </div>
+          <div class="intro-container-1">
+            <div class="intro-description">
+              <p>
+                I'm a software developer! I can help you build a product,
+                feature or website. Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
+            </div>
+          </div>
+          <div class="connect">
+            <p>LET'S CONNECT</p>
```

Comment:
Kindly update this element to use a `<hx>` tag as it is a subtitle🙏

Reply:
fixed